src/seedu/addressbook/data/person/Person.java

```diff
@@ -3,7 +3,8 @@
 import java.util.HashSet;
 import java.util.Objects;
 import java.util.Set;
-
+import java.util.Comparator;
+import java.util.*;
```

Comment:
Coding standard violation! This might be due to your IntelliJ settings, so please check your IDE's settings.